components/MermaidDiagram.module.css

```
@@ -38,8 +36,11 @@
  */
 .mermaidTarget :global(foreignObject > div),
 .mermaidTarget :global(.nodeLabel),
-.mermaidTarget :global(.edgeLabel) {
+.mermaidTarget :global(.edgeLabel),
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)